Dockerfile

```diff
@@ -650,11 +668,57 @@ RUN chmod +x /installer
 FROM alpine:3.18.0 AS unicode-pf2
 RUN apk add --no-cache --update --no-scripts grub
 
+FROM --platform=${BUILDPLATFORM} tools-prepare AS uki-build-amd64
+WORKDIR /build
+COPY hack/certs certs
+COPY hack/uki .
+COPY --from=pkg-sd-stub-amd64 / .
+COPY --from=embed /pkg/machinery/gendata/data data
+COPY --from=pkg-kernel-amd64 /boot/vmlinuz vmlinuz
+COPY --from=initramfs-archive-amd64 /initramfs.xz initramfs.xz
+RUN <<EOF
+cat > os-release <<EOFINNER
+NAME="$(cat data/name)"
+ID=$(cat data/name)
+VERSION_ID=$(cat data/tag)
+PRETTY_NAME="$(cat data/name) ($(cat data/tag))"
+HOME_URL="https://www.talos.dev/"
+BUG_REPORT_URL="https://github.com/siderolabs/talos/issues"
+EOFINNER
+EOF
+RUN --security=insecure mknod /dev/tpmrm0 c 10 224 && \
```

Comment:
currently requires access to a tpm, figure out if we can use a virtual emulated one